src/components/AdminSettings/SignalingServer.vue

```diff
@@ -77,6 +80,54 @@ export default {
 		},
 	},
 
+	data() {
+		return {
+			checked: false,
+			customError: '',
+			versionFound: '',
+			canNotConnect: false,
+			responseNotValidJson: false,
+			versionNotSupported: false,
```

Comment:
As `canNotConnect`, `responseNotValidJson` and `versionNotSupported` are only used to show one error message or the other I would just remove those attributes for now and store the appropriate error message in `customError` (probably renaming it to `errorMessage` or something like that).

Reply:
done